Importer/CsvFileImporter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Netdudes\ImporterBundle\Importer;
+
+class CsvFileImporter extends CsvImporter
+{
+    /**
+     * @param string $filename
+     * @param bool   $dryRun
+     */
+    public function import($filename, $dryRun = false)
```

Comment:
The interface uses `$data`, here we have `$filename`, on parent we have `$csv`.
Crappy php...